Cargo.toml

```diff
@@ -114,6 +114,9 @@ unused_lifetimes = "warn"
 trivial_casts = "allow"
 unused_qualifications = "allow"
 
+# MEMBRANE: added to suppress warning
+unexpected_cfgs = { level = "warn", check-cfg = ['cfg(web_sys_unstable_apis)'] }
+
```

Comment:
Nice! I didn't know you could suppress warnings here 👍

Reply:
The Rust compiler had a helpful message after the warning suggesting it :)